tests/framework/types.go

```diff
@@ -39,22 +44,22 @@ type VirtualMachine struct {
 
 func (vm VirtualMachine) Create() (string, string, error) {
 	args := []string{"create", "-f", vm.Manifest}
-	return ktests.RunCommandWithNS(vm.Namespace, ktests.KubeVirtOcPath, args...)
+	return ktests.RunCommandWithNS(vm.Namespace, ocName, args...)
 }
 
 func (vm VirtualMachine) Start() (string, string, error) {
 	args := []string{"start", vm.Name}
-	return ktests.RunCommandWithNS(vm.Namespace, ktests.KubeVirtVirtctlPath, args...)
+	return ktests.RunCommandWithNS(vm.Namespace, ocName, args...)
```

Comment:
oc don't start VM, virtctl is.
and  like Lukas comment please use the path exist already.